src/core/handle.ml

```diff
@@ -185,7 +194,35 @@ let handle_cmd : sig_state -> p_command -> sig_state * proof_data option =
       (* Actually add the symbol to the signature. *)
       out 3 "(symb) %s ≔ %a\n" x.elt pp_term t;
       let d = if op then None else Some(t) in
-      let ss = Sig_state.add_symbol ss e Defin x a impl d in
+      let (ss, _) = Sig_state.add_symbol ss e Defin x a impl d in
+      (ss, None)
+  | P_inductive(e, s, t, tl)     ->
+      (* Add the head symbol in the signature *)
+      let (ss, typ) = handle_symbol ss e Const s [] t in
+      (* To create the body of the inductive type *)
+      let add_cons :
+                sig_state * sym list -> ident * p_term -> sig_state * sym list
+        = fun (ss, cons) (id, a) ->
+        let (ss, c) = handle_symbol ss e Injec id [] a in
+        (ss, c::cons)
+      in
+      (* Add the inductive type in the signature *)
+      let (ss, cons) = List.fold_left add_cons (ss, []) tl in
+      (* Compute the inductive principle *)
+      let pr = Inductive.principle typ cons in
+      let path = (current_sign()).sign_path in
+      let pr =  { sym_name = ((typ.sym_name)^"_ind")
```

Comment:
You can simply write `typ.sym_name ^ "_ind"`.

Reply:
Try to not create symbol structures directly. Instead you can write:
```
let ind_sym = Sign.add_symbol ss.signature e cmd.pos pr [] in
```

Reply:
OK, but this code it's just a "glue code".
I'm deleting this record.